### DISCUSSION TITLE
Put newcommand tex file after newcommands in main tex

### DISCUSSION THREAD

**Discussion by Turgon-Aran-Gondolin** [James-Yu/LaTeX-Workshop#3121] [Q&A]
**Is your feature request related to a problem? Please describe.**
When I use preview feature, I found the commands I defined in the newcommand tex file got overwrited by the newcommands from the main tex file. This cause the preview to be unstable, even break down in most cases. 

**Describe the solution you'd like**
Add the new command tex file after the new commands from the main file, but before the actual math environment. This way I think I can define newcommand in the global new command file to overwrite the ones from my main tex. I don't know if two conflicting new command definition will cause further problems. 

**Describe alternatives you've considered**
I don't have another idea to avoid the conflict between these two.

**Comment by tamuratak** [comment]
- Please paste sample TeX files.
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.
